### PR TITLE
add ignore support

### LINKED ISSUE #5: Loading subdirs
- `app/controllers/subdir/haha/foo.js` => `app.controllers.subdir.haha.foo`
- glob `*/*.js`
- remove extra
- remove `exports.init(app)` and `module.exports = function (app)`

### LINKED ISSUE #15: 支持ignore配置
自动载入services包得时候，有时候services下需要util，这时候只能把util放到和services目录同级了。

但是util只是为services服务的，这时候最好还是util放在services目录下。

我觉得增加ignore配置，或者默认把util目录忽略不load了。

### PR DESCRIPTION
**Pull request by shepherdwind**
Closes #15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "debug": "~2.1.3",
-    "glob": "~4.0.5",
+    "globby": "^2.0.0",
```
Comment (popomore) [inline package.json]: use ~

File: README.md [hunk 2]
```diff
@@ -89,6 +89,11 @@ loading(PATH,[{call:true}]).into(TARGET, FIELD, [filters])
   decide whether override the property that is already exist,
   default: false
 
+- **ignore** type:[`string function`]
+  ignore file. If ignore is string, ignore role will be glob string `!ignore`,
```
Comment (popomore) [inline README.md]: I think string is enough, glob support many patterns. and you can support array that can get multi glob.

File: package.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "debug": "~2.1.3",
-    "glob": "~4.0.5",
+    "globby": "~2.0.0",
```
Comment (shepherdwind) [inline package.json]: @popomore change to ~ now.

**Comment by shepherdwind** [comment]
Travis-ci fail at node 0.11, but Than run ok at my compute.

The fail part is just the array order changed, Maybe we should just ignore 0.11 error, just run on 0.12.

**Comment by fengmk2** [comment]
@shepherdwind remove 0.11 from travis ci

File: .travis.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - '0.11'
-  - '0.10'
+  - '2.3'
```
Comment (fengmk2) [inline .travis.yml]: iojs-2

**Comment by shepherdwind** [comment]
@fengmk2 I have remove 0.11 and 0.12 travis  runner. Only left 0.10 and iojs-2.

**Comment by popomore** [comment]
LGTM

**Comment by popomore** [comment]
change 0.11 to 0.12?

**Comment by fengmk2** [comment]
+1

**Comment by fengmk2** [comment]
@shepherdwind 1.7.0

**Pull request merged**